requirements.txt

```
@@ -1,2 +1,2 @@
-karton-core==4.0.5
-yara-python==4.0.2
+karton-core>=4.2.0
```

```suggestion
karton-core>=4.2.0,<5.0.0
```
We've decided that we'd like to have this safety net in case we decide to introduce any compatiblity-breaking changes